src/queries/__tests__/lpFills.test.ts

```diff
@@ -113,4 +114,59 @@ describe('getLpFills', () => {
       ]
     `);
   });
+
+  it('should correcvtly sort lpFills by filledAmountValueUsd in descennding order', async () => {
+    vi.spyOn(lpClient, 'request').mockResolvedValueOnce({
+      limitOrders: {
+        groupedAggregates: [
+          {
+            keys: ['1'],
+            sum: {
+              filledAmountValueUsd: '1',
+            },
+          },
+          {
+            keys: ['2'],
+            sum: {
+              filledAmountValueUsd: '1',
+            },
+          },
+          {
+            keys: ['3'],
+            sum: {
+              filledAmountValueUsd: '3',
+            },
+          },
+        ],
+      },
+      rangeOrders: {
+        groupedAggregates: [
+          {
+            keys: ['2'],
+            sum: {
+              baseFilledAmountValueUsd: '1',
+              quoteFilledAmountValueUsd: '1',
+            },
+          },
+          {
+            keys: ['3'],
+            sum: {
+              baseFilledAmountValueUsd: '0.5',
+              quoteFilledAmountValueUsd: '0.5',
+            },
+          },
+        ],
+      },
+    });
+
+    const start = '2024-10-31T00:00:00Z';
+    const end = addWeeks(start, 1).toISOString();
+    const result = await getLpFills({ start, end });
+
+    const sorted = [...result].sort((a, b) =>
```

Comment:
Just found out that we have to make a copy of an array to ensure that `toEqual ` will check the order of objects, since somehow `sort` function prevents from checking the order 🤔

Reply:
Yup, genau 😉